README.md

```diff
@@ -23,6 +23,66 @@ Minor modifications were made in order to get sct to:
 
 # Installation
 
+## Make-based
+
+On UNIX-based systems, the most convenient method of building this software is
+using Make. Since the `Makefile` currently is simplistic enough, both the BSD
```

Comment:
```suggestion
using [make(1)](https://www.gnu.org/software/make/). Since the `Makefile` is simple and portable, both the BSD
```

Reply:
Half NAK on that. I agree with the second part, but the first one is better to stay `Make`. I'm talking about the program family there, not a specific implementation like GNU make. I will, however, add the link to the GNU make part.

Reply:
Okay, this makes sense.